### DISCUSSION TITLE
I am worried about Volkan Ceylan

### DISCUSSION THREAD

**Discussion by moseur** [serenity-is/Serenity#6677] [General]
Hello Dear All,

I am worried about Turkey Earth quake. Worried About Volkan Ceylan. Hey Dear Volkan Ceylan, are you Okay?? I don't have your personals contact that's why I am here. 

May Allah save you, save your family, also save your country. 
Allah please help us.

Thanks 
Moseur

**Comment by edwardch** [comment]
Thinking of all the people of Turkey during this extremely difficult time.

Sending you all strength.

**Comment by wezmag** [comment]
I am also worried about them. 
But, I think they should be fine. 
Istanbul is quite far from the epicenter.
However, they must be very busy helping others in Turkiye.
If there is anything we can do, please let us know.

**Comment by VictorTomaili** [comment]
Hello everyone, thank you so much. We are located in Istanbul and we're all safe. But there is a lot of people have hard time in east of country. Thanks again for all the wishes and support.

**Reply by DrastiGohel** [reply]
Good to know you all are safe. Hope everything gets better.

**Reply by moseur** [reply]
Thanks Allah, Thank you for your kind reply. Its really very hard time for Turkey & Seria. But we are nothing in front of nature. 

May Allah give strength to all victims. 

Thanks again.

**Comment by MimmoDany** [comment]
A big hug to all of you.
In Italy we are doing everything possible to lend a hand to the Turkish people.
I hope you are all well and your loved ones are staying safe.

**Comment by gfo2007** [comment]
It is a sad time for the world and especially for Turkey. I am a Ghanaian living in Ghana. Just yesterday, it was confirmed that our national footballer who plays in Turkey, Christian Atsu has finally passed on after he was trapped during the earthquake. May his soul and that of all the departed rest in peace. I am very sad and so are many Ghanaians.

We pray for safety for the rest of Turkey and hope God spares from these natural disasters. 

Be safe y'all.
